Client/keycloak.json

```diff
@@ -0,0 +1,8 @@
+{
+  "realm": "TACT",
+  "auth-server-url": "http://localhost:8180/realms/TACT",
```

Comment:
@poppyjflow is this going to cause problems later when this goes into prod? Or we will use someone else's auth-server?